functions/defaulted_parameters.md

```diff
@@ -0,0 +1,87 @@
+# Module name: topic name
+_Skeleton instructions are typeset in italic text._
+
+## Overview
+
+Functions in C++ may be overloaded with different numbers and types of parameters. It may be of value to specify default values for some number of parameters, to allow a consumer to avoid specifying parameters that rarely change, or to enable expanding the set of parameters while maintaining backward compatibility with existing consumers.
+
+<table>
+  <thead>
+    <th>Level</th>
+    <th>Objectives</th>
+  </thead>
+  <tr>
+    <td>Foundational knowledge</td>
+    <td>can make calls to functions with or without defaulted parameters</td>
+  </tr>
+  <tr>
+    <td>Advanced</td>
+    <td>declares and defines functions with defaulted parameters</td>
+  </tr>
+  <tr>
+    <td>Further studies</td>
+    <td>refinement of defaulted parameters through multiple declarations</td>
+  </tr>
+</table>
+
+## Motivation
+
+Default parameters allow the omission of parameters with obvious or common values. Also may be utilized to extend an existing function signature without forcing changes to existing calling code.
+
+## Topic introduction
+
+Explain how default parameters work and how to define them.
+
+## Foundational knowledge: Using *
```

Comment:
`Using a function with default arguments`

And honestly, is there anything at all to teach here? Calling a function with or without default arguments looks and works the same way: `foo(some, args, here)`. I think "Defining a function with default arguments" should be included in the "Basic" part, and maybe there's just no "Advanced" part at all (except for teaching [the vast number of caveats that militate against ever using default arguments](https://quuxplusone.github.io/blog/2020/04/18/default-function-arguments-are-the-devil/)).

Reply:
That could very well be. I'd be happy to move all the things to "basic". Admittedly, I put them under "advanced" in order to have _something_ under advanced.

Reply:
done